python/pyarrow/table.pxi

```diff
@@ -397,6 +397,13 @@ cdef class ChunkedArray(_PandasConvertible):
         """
         return _pc().take(self, indices)
 
+    def drop_null(self):
+        """
+        Remove missing values from a chunked array.
+        See pyarrow.compute.drop_null for full usage.
```

Comment:
"for full description" perhaps, because the function signature (hence usage) is trivial?